src/components/TitleBanner/TitleBanner.module.scss

```diff
@@ -1,16 +1,28 @@
 @use "utils/bootstrap";
 @import "styles/hackuci-overrides";
 
+$banner-dark-blue: #153d63;
+$banner-lit-blue: #3a6895;
+
 .titleBanner {
-	width: calc(220px + 35vw);
-	max-width: 800px;
+	--banner-opacity: 0.9;
+
+	width: calc(200px + 35vw);
+	max-width: 600px;
 	position: relative; // to absolutely position edges
 
+	margin-bottom: calc(4rem + 4.25vw - 21vh);
+	margin-top: 8vh;
+	margin-left: auto;
+	margin-right: auto;
+
 	.lamp {
 		position: absolute;
 		margin: auto;
-		top: -8%;
+		top: -10%;
```

Comment:
Not exactly an ideal solution, but to address Sam's comment, we can change this to a non-percentage value so at least it seems attached to the ceiling 🤷 (I tried with -6vh)

Reply:
Thanks for the thought, maybe something we could fix later.